emulators/qemu/Portfile

```diff
@@ -8,7 +8,7 @@ PortGroup legacysupport 1.1
 legacysupport.newest_darwin_requires_legacy 16
 
 name                    qemu
-version                 7.1.0
+version                 7.2.0
 revision                1
```

Comment:
Also `revision` resets to zero whenever the `version` gets updated:

```suggestion
revision                0
```

Reply:
fixed